Jenkinsfile

```
@@ -1,16 +1,31 @@
 @Library('ctsrd-jenkins-scripts') _
 
-// Set the default job properties (work around properties() not being additive but replacing)
-setDefaultJobProperties([rateLimitBuilds([count: 1, durationName: 'hour', userBoost: true]),
-                         [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/CTSRD-CHERI/cheribsd/'],
-                         copyArtifactPermission('*'), // Downstream jobs need the kernels/disk images
-])
+class GlobalVars { // "Groovy"
+    public static boolean archiveArtifacts = false;
+}
 
 if (env.CHANGE_ID && !shouldBuildPullRequest()) {
     echo "Not building this pull request."
     return
 }
 
+// Set job properties:
+def jobProperties = [rateLimitBuilds([count: 1, durationName: 'hour', userBoost: true]),
+                     [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/CTSRD-CHERI/cheribsd/'],
+                     copyArtifactPermission('*'), // Downstream jobs need the kernels/disk images
+]
+// Don't archive sysroot/disk image/kernel images for pull requests and non-default branches:
+def archiveBranches = ['master', 'dev']
+if (/* !env.CHANGE_ID && archiveBranches.contains(env.BRANCH_NAME) */ true) {
```

```suggestion
if (!env.CHANGE_ID && archiveBranches.contains(env.BRANCH_NAME)) {
```
Change before merging